br/pkg/restore/db.go

```diff
@@ -90,6 +99,10 @@ func (db *DB) ExecDDL(ctx context.Context, ddlJob *model.Job) error {
 		return errors.Trace(err)
 	}
 
+	if ddlJob.Query == "" {
+		return nil
```

Comment:
please add a warn log to record this DDL. something like DDL type

Reply:
done